crates/libcontainer/src/syscall/linux.rs

```diff
@@ -279,6 +283,11 @@ impl Syscall for LinuxSyscall {
             errno
         })?;
 
+        close(newroot).map_err(|errno| {
```

Comment:
Is this explicit close required? Does the `O_CLOEXEC` take care of closing after the function returns, or there is a specific reason the fd will not be closed unless we explicitly call close?